src/fft.cpp

```diff
@@ -63,7 +37,7 @@ namespace FFT
 
         const uint32_t size = image.width() * image.height();
 
-        _data = reinterpret_cast<kiss_fft_cpx *>( malloc( size * sizeof(kiss_fft_cpx) ) );
+        _allocateData(size * sizeof(kiss_fft_cpx)); 
```

Comment:
There is a trailing whitespace. Please remove it as analysers would complain about it. If you use Visual Studio you can use 2 combination to do full formatting (with proper setup for C++ code): press `Ctrl+K` and the press `Ctrl+D`. This should format a whole text. I made 25 commits to fix formatting issues.

Reply:
Done.